### ISSUE TITLE
Is it possible to run SlicerTMS without diffusion MRI.

### ISSUE THREAD

**Issue by jmz3** [lorifranke/SlicerTMS#16]
Hi! Thanks for your nice work.

I would like to ask whether it's OK not to have the diffusion MRI and the connectivity map. If I'm only interested in the E field distribution and I only have the structured MRI for the subject, nothing else, can I just replace the gray matter and skin mesh file and keep the rest as is in the example data? I think the underlying problem is whether the model predicts the E field solely according to the GM, Coil and the M field, or it relies on the connectivity map as well.

Thank you in advance!

**Comment by lorifranke** [comment]
@jmz3 Yes of course, you can just replace the gray matter and skin mesh file and keep the rest from the example folder. In the UI, you can then hide the fibers/tractography. However, you will still need to download the SlicerDMRI extension in 3D Slicer (sorry for that).
